From 8338f902592105d44d01719016dbcecd0e92fe7e Mon Sep 17 00:00:00 2001
From: Ondrej Jirman <megous@megous.com>
Date: Mon, 18 Oct 2021 17:41:25 +0200
Subject: [PATCH 378/446] arm64: dts: sun50i-a64-pinephone: Add supply for i2c
 bus to anx7688

The bus power needs to be enabled for anx7688 to be able to communicate
with the master.

Signed-off-by: Ondrej Jirman <megous@megous.com>
---
 arch/arm64/boot/dts/allwinner/sun50i-a64-pinephone-1.0.dts | 1 +
 arch/arm64/boot/dts/allwinner/sun50i-a64-pinephone-1.1.dts | 1 +
 arch/arm64/boot/dts/allwinner/sun50i-a64-pinephone-1.2.dts | 1 +
 3 files changed, 3 insertions(+)

diff --git a/arch/arm64/boot/dts/allwinner/sun50i-a64-pinephone-1.0.dts b/arch/arm64/boot/dts/allwinner/sun50i-a64-pinephone-1.0.dts
index a21c6d78a..062532416 100644
--- a/arch/arm64/boot/dts/allwinner/sun50i-a64-pinephone-1.0.dts
+++ b/arch/arm64/boot/dts/allwinner/sun50i-a64-pinephone-1.0.dts
@@ -43,6 +43,7 @@ hdmi-bridge@28 {
 		avdd18-supply = <&reg_ldo_io1>;
 		dvdd18-supply = <&reg_ldo_io1>;
 		avdd33-supply = <&reg_dldo1>;
+		i2c-supply = <&reg_ldo_io0>;
 		vconn-supply = <&reg_vconn5v0>;
 		hdmi_vt-supply = <&reg_dldo1>;
 
diff --git a/arch/arm64/boot/dts/allwinner/sun50i-a64-pinephone-1.1.dts b/arch/arm64/boot/dts/allwinner/sun50i-a64-pinephone-1.1.dts
index 63ceae07a..35cb92e3b 100644
--- a/arch/arm64/boot/dts/allwinner/sun50i-a64-pinephone-1.1.dts
+++ b/arch/arm64/boot/dts/allwinner/sun50i-a64-pinephone-1.1.dts
@@ -52,6 +52,7 @@ hdmi-bridge@28 {
 		avdd18-supply = <&reg_ldo_io1>;
 		dvdd18-supply = <&reg_ldo_io1>;
 		avdd33-supply = <&reg_dldo1>;
+		i2c-supply = <&reg_ldo_io0>;
 		vconn-supply = <&reg_vconn5v0>;
 		hdmi_vt-supply = <&reg_dldo1>;
 
diff --git a/arch/arm64/boot/dts/allwinner/sun50i-a64-pinephone-1.2.dts b/arch/arm64/boot/dts/allwinner/sun50i-a64-pinephone-1.2.dts
index 613d41bb1..61af73e88 100644
--- a/arch/arm64/boot/dts/allwinner/sun50i-a64-pinephone-1.2.dts
+++ b/arch/arm64/boot/dts/allwinner/sun50i-a64-pinephone-1.2.dts
@@ -55,6 +55,7 @@ hdmi-bridge@28 {
 		avdd18-supply = <&reg_ldo_io1>;
 		dvdd18-supply = <&reg_ldo_io1>;
 		avdd33-supply = <&reg_dcdc1>;
+		i2c-supply = <&reg_ldo_io0>;
 		vconn-supply = <&reg_vconn5v0>;
 		hdmi_vt-supply = <&reg_dldo1>;
 
-- 
2.31.1

